### PR TITLE
Clean scala compiler

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala [hunk 1]
```diff
@@ -138,20 +132,22 @@ case class MacDistHandler(jre: Option[String], jreName: String)
...
           override def visitFile(
-              file: Path,
-              attrs: BasicFileAttributes
+            file:  Path,
+            attrs: BasicFileAttributes
           ): FileVisitResult = {
+            if (file.toFile.getName.startsWith("scala-compiler")) {
+              println(s"Remove native libs from scala-compiler jar")
+              val cmd = Seq("zip", "-d", file.toFile.getAbsolutePath, """META-INF/native/**""" )
+              cmd.lines.foreach(println)
```
Comment (swalker2m) [inline project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala:141]: Where does it actually do the removal?

Comment (cquiroz) [reply project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala:141]: When you call `cmd.lines` it  executes and produces an output

Comment (swalker2m) [reply project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala:141]: Seems to be a method on `ProcessBuilder`, I wonder how `Seq` becomes a `ProcessBuilder`.  But I don't wonder enough to investigate. :+1:

**Review by swalker2m — APPROVED** [review]

File: project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala (hunk 1 above)
Comment (cquiroz) [reply project/src/main/scala/edu/gemini/osgi/tools/app/MacDistHandler.scala:141]: It is syntax added on the scala lib
https://www.scala-lang.org/api/2.12.8/scala/sys/process/index.html

**Pull request merged**